docs/graph-algorithms/breadth-first-search.mdx

```diff
@@ -0,0 +1,264 @@
+---
+title: Breadth First Search
+slug: /graph-algorithms/breadth-first-search
+---
+
+## Introduction
+Breadth-first search (BFS) is an algorithm for traversing or searching tree or graph data structures.
+It starts at the tree root (or some arbitrary node of a graph, sometimes referred to as a 'search key'[1]),
+and explores all of the neighbor nodes at the present depth prior to moving on to the nodes at the next depth level.
+
+## Explanation
+
+- Given an undirected & Connected Graph (V, E)
+  V: Vertices E: Edges
+- Goal: methodically explore every vertex and every edge
+- “Explore” a graph, turning it into a tree
+- One vertex at a time
+- Expand frontier of explored vertices across its breadth
+- Discovers all vertices at distance k from s before those at distance k+1
+- A level-by-level order traversal
+- For example, in the following graph, we start traversal from vertex 2. When we come to vertex 0, we look for all adjacent vertices of it. 2 is also an adjacent vertex of 0. If we don’t mark visited vertices, then 2 will be processed again and it will become a non-terminating process.
+ A Breadth First Traversal of the following graph is 2, 0, 3, 1.
+ [![graph.png](https://media.geeksforgeeks.org/wp-content/uploads/bfs-5.png)]
+
+##Applications of Breadth First Traversal
```

Comment:
it should be `## Applications of....` and not `##Applications of....`.
You are missing a space.